README.md

```diff
@@ -83,6 +88,18 @@ The commands are:
 Use "fyne-cross <command> -help" for more information about a command.
 ```
 
+### Apple Arm Silicon special
+
+On Apple Arm64 architecture, the container have to run in a vm. This vm put some constraint on getting fyne-cross to work in that setup.
```

Comment:
```suggestion
On Apple Arm64 architecture, the container has to run in a vm. This vm puts some constraints on getting fyne-cross to work with that setup.
```

Reply:
Oh, forgot we had some stuff in the README regarding Apple Arm64. I have clarified that now as it was inaccurate.